### DISCUSSION TITLE
EU Regions - map transitions to scatterplot (MAUP)

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/interactivity#6] [Cool Scrolly Ideas]
link: https://pudding.cool/2019/04/eu-regions/

- Really cool transition between map and scatter plot.
- conveys MAUP ? ahha right @usamabilal ?
- the transition between map and scatter plot: really conveys a lot of information particularly allows us to make conecct spatial distribution and patterns in these distributions. 
----

![image](https://user-images.githubusercontent.com/45013044/229820274-5632a5e7-1be3-4479-a366-707a27069ff4.png)

**Comment by ran-codes** [comment]
@usamabilal I think this is sone of the priority components for our library. i think this transition is really interesting and valuable and adapatbale to so many UHC/SALURBAL/PDPH use cases. 

This viz is also open source. The source code is here: https://github.com/the-pudding/eu-regions

We will evaluate the source code and @anujtanwar98 can start adapting later this month after we have basic D3/Svelte components/knowledge ready.

**Comment by usamabilal** [comment]
Fantastic! This is indeed one of those we can use for MANY things both locally and for SALURBAL, so lets try to replicate this. 
(and yes, this is MAUP!). We could use this for @alinasmahl1's redistricting paper too in case we need another case study

**Reply by ran-codes** [reply]
side thought. another potential component is one dedicated for congressional district visualizations... as there aren't much using CD boundaries.

**Reply by ran-codes** [reply]
cool bounadries (hexagons) i saw https://www.dailykos.com/stories/2015/06/03/1389806/-Daily-Kos-Elections-presents-the-best-map-ever-of-United-States-congressional-districts. these are also open source and available for us to adapt

**Reply by usamabilal** [reply]
@alinasmahl1 interesred in pursuing this?

**Reply by usamabilal** [reply]
> cool bounadries (hexagons) i saw https://www.dailykos.com/stories/2015/06/03/1389806/-Daily-Kos-Elections-presents-the-best-map-ever-of-United-States-congressional-districts. these are also open source and available for us to adapt

the thing is that the ACTUAL boundaries are important in this case, not so much hexagons. because we want ot show gerrymandering!

**Reply by ran-codes** [reply]
ahhh. got it. yea... gerrymandered boundaries are wild.... and indeed a huge part of the story.

**Reply by alinasmahl1** [reply]
but keep reading the article Ran linked-- their map actually shows the gerrymandering really well. it's the 1-2 blue districts surrounded by red.

**Reply by alinasmahl1** [reply]
@usamabilal but yes, i thikn interesting to potentially pursue! could be the addition we need for the redistricting article?

**Reply by usamabilal** [reply]
> @usamabilal but yes, i thikn interesting to potentially pursue! could be the addition we need for the redistricting article?

exactly wha im thinking, to explain the gerrymandering mechanism

**Reply by usamabilal** [reply]
@ran-codes it'd be great to organize a sort of "project" type of label so we can start groping stuff for the differnt ideas:
- congressional districts
- life expectancy
- pricila's RWJF one
- a SALURBAL one (excess mortality?)
(or i am sure you already know how to keep this organized in the repo!)
